evaluation/Aggregation.py

```diff
@@ -56,9 +56,12 @@ def run_agg_query(self, df, metadata_path, query, confidence):
         metadata = MetadataLoader(metadata_path).read_schema()
```

Comment:
the files with .py should be snake cased to follow python convention. It is not introduced here but evaluation/aggregation.py would be preferred. 

FYI In case a rename is attempted. It is sometimes non trivial to rename a file if only case sensitivity is changed due to file system nuances.

Reply:
This helped me rename - 
git mv previous_file_name new_file_name